src/bijectors/logit.jl

```diff
@@ -38,4 +38,4 @@ logabsdetjac(b::Logit{2}, x::AbstractMatrix) = sum(logit_logabsdetjac.(x, b.a, b
 logabsdetjac(b::Logit{2}, x::AbstractArray{<:AbstractMatrix}) = map(x) do x
     logabsdetjac(b, x)
 end
-logit_logabsdetjac(x, a, b) = -log((x - a) * (b - x) / (b - a))
\ No newline at end of file
+logit_logabsdetjac(x, a, b) = -log((x - a) * (b - x) / (b - a))
```

Comment:
Since this showed up in the commits, is there a reason for not writing
```suggestion
logit_logabsdetjac(x, a, b) = log(b - a) - log(x - a) - log(b - x)
```
?

Reply:
Hmm on second thought, I think your version has 3 log calls so it will be slower than the first one which has a single log. `*` and `/` are generally faster than `log`. For example:
```julia
julia> using BenchmarkTools

julia> f(x, a, b) = -log((x - a) * (b - x) / (b - a))
f (generic function with 1 method)

julia> g(x, a, b) = -log(x - a) - log(b - x) + log(b - a)
g (generic function with 1 method)

julia> fb(x, a, b) = f.(x, a, b)
fb (generic function with 1 method)

julia> gb(x, a, b) = g.(x, a, b)
gb (generic function with 1 method)

julia> @btime fb($(rand(1000)), 0.0, 1.0);
  8.866 μs (1 allocation: 7.94 KiB)

julia> @btime gb($(rand(1000)), 0.0, 1.0);
  18.699 μs (1 allocation: 7.94 KiB)
```
In this case, we are only multiplying 2 numbers so underflow is unlikely. Hence there is no need for the log sum trick.